### PR TITLE
show correct info text if no filtered entries were found

### PR DESCRIPTION
**Pull request by jagaSto**
* fixes https://github.com/kenodressel/quarantine-hero/issues/16 --> currently, the info text is not shown if the search returned no results
* updated info text (removed redundant word)
* autoformatting moved some stuff :D

Screenshot 2020-03-18 at 21 37 16

### CODE REVIEW

File: src/components/FilteredList.js [hunk 1]
```diff
@@ -66,19 +66,20 @@ export default function FilteredList() {
...
+    <div className="pt-3">
+      <LocationInput onChange={handleChange} value={location} onSelect={handleSelect} />
+    </div>
+    <div className="py-3 w-full">
+      <div className="my-3 w-full">
+        <Link to='/notify-me' className="btn-green-secondary my-3 mb-6 w-full block" onClick={() => fb.analytics.logEvent('button_subscribe_region')}>
+          Benachrichtige mich wenn jemand in {location && location !== '' ? `der Nähe von ${location}` : 'meiner Nähe'} Hilfe braucht!</Link>
       </div>
+      {
+        filteredEntries.length === 0
```
Comment (jagaSto) [inline src/components/FilteredList.js]: the actual change: changed `entries` to `filteredEntries` (entries would never be empty)

**Comment by florianschmidt1994** [comment]
Thanks for contributing! I'll check it out :)

**Comment by florianschmidt1994** [comment]
Works as intended! Could you please make sure to only have the code changes on this branch and remove the changed code styling? This will make merging other PRs in the future easier for us :)

**Comment by jagaSto** [comment]
sure, let me open a new PR real quick to get rid of this messy commit history!

**Pull request closed without merging**